cnf-certification-test/platform/cnffsdiff/fsdiff.go

```diff
@@ -73,6 +74,17 @@ func NewFsDiffTester(client clientsholder.Command) *FsDiff {
 	}
 }
 
+func intersectTargetFolders(src []string) []string {
+	var dst []string
+	for _, folder := range src {
+		if targetFolders.Contains(folder) {
```

Comment:
Instead of this, I believe we could just use the existing `StringInSlice` func found [here](https://github.com/test-network-function/cnf-certification-test/blob/main/pkg/tnf/utils/utils.go#L25).  That way we don't need the MapSet library at all.

Reply:
Decided against switching this to use StringInSlice to take advantage of the O(1) lookup instead of the O(n).

Reply:
Also yes, we know that the old Go maps were O(1) as well.  😄